Cargo.toml

```
@@ -13,7 +13,10 @@ categories = ["asynchronous", "concurrency"]
 readme = "README.md"
 
 [dependencies]
-futures = { version = "0.3.4", default-features = false, features = ["std"] }
+futures-util = { version = "0.3.5", default-features = false, features = ["std"] }
```

```suggestion
futures-util = { version = "0.3.5", default-features = false, features = ["std", "sink"] }
```

In futures-util, [sink is not enabled by default](https://github.com/rust-lang/futures-rs/blob/master/futures-util/Cargo.toml#L22).

([CI passes because of the feature flag of the dev-dependencies leaks to normal dependencies.](https://github.com/rust-lang/cargo/issues/4866))